zcash_client_backend/src/data_api.rs

```diff
@@ -1294,7 +1294,8 @@ pub trait InputSource {
         account: Self::AccountId,
         target_value: TargetValue,
         sources: &[ShieldedProtocol],
-        anchor_height: BlockHeight,
+        target_height: zcash_protocol::consensus::TargetHeight,
```

Comment:
There's something about the `delegatable_trait` attribute macro that is hiding the import of `TargetHeight`. I'm not sure why `BlockHeight` isn't affected. Maybe this is just on my machine? Anyway - that's why I'm qualifying here.

Reply:
What's happening here is likely to be related to feature flagging.